cluster/v1/types.go

```diff
@@ -84,7 +84,7 @@ type ClientConfig struct {
 // ManagedClusterStatus represents the current status of joined managed cluster.
 type ManagedClusterStatus struct {
 	// Conditions contains the different condition statuses for this managed cluster.
-	Conditions []StatusCondition `json:"conditions"`
+	Conditions []metav1.Condition `json:"conditions"`
```

Comment:
be sure your current data is compatible with this change

Reply:
yes, i am quite sure it is compatible.